spring-xd-dirt/src/main/java/org/springframework/xd/dirt/server/MessageBusExtensionsConfiguration.java

```diff
@@ -43,13 +44,16 @@
 public class MessageBusExtensionsConfiguration {
 	@Autowired
 	ApplicationContext applicationContext;
+	
+	@Value("${spring.xd.codec.kryo.references:true}")
```

Comment:
May be we need servers.yml entry for this? also, instead of specifying the default value here, can we define it in application.yml?